CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/reporting/GraphReportManager.java

```diff
@@ -18,46 +18,24 @@
 import au.gov.asd.tac.constellation.graph.Graph;
 import au.gov.asd.tac.constellation.graph.manager.GraphManager;
 import au.gov.asd.tac.constellation.graph.manager.GraphManagerListener;
-import au.gov.asd.tac.constellation.utilities.text.SeparatorConstants;
 import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
 import java.util.Collections;
 import java.util.HashMap;
-import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
-import java.util.Set;
-import java.util.TreeSet;
-import java.util.prefs.Preferences;
-import org.openide.util.NbPreferences;
 
 /**
  *
  * @author sirius
  */
 public class GraphReportManager {
 
-    private static final String IGNORED_TAGS_KEY = "reportManagerIgnoredTags";
-
     private static final List<GraphReportListener> LISTENERS = new ArrayList<>();
 
     private static final Map<String, GraphReport> GRAPH_REPORTS = Collections.synchronizedMap(new HashMap<>());
 
-    /**
-     * Tags that will be ignored (not recorded) by GraphReport instances.
-     */
-    private static final Set<String> IGNORED_TAGS;
 
     static {
-        // Determine the ignored tags *before* we set up the graph reports.
-        final Preferences prefs = NbPreferences.forModule(GraphReportManager.class);
-        final String filteredTagString = prefs.get(IGNORED_TAGS_KEY, "LOW LEVEL").trim();
```

Comment:
I'm wondering if we can retain the filtered tags logic code in the Manager class? I see that this code has now moved to the `PluginReporterPane` which is part of the Views. Given we have the Notes View also responding to the `GraphReportManager` I feel that in terms of responsibility it should be the manager that gets to initialise the filter list and let the views know. What do you think?

Reply:
Ignore me, you haven't moved it. I think something worth investigating later on is if more control should be given to the `GraphReportManager`.